invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/EndorsementsDisplay.js

```diff
@@ -110,15 +43,15 @@ export class EndorsementsDisplay extends Component {
     })).sort((a, b) => b.index - a.index);
   };
 
-  getVersion = (review, versions, latestVersion) => {
+  const getVersion = (review, versions, latestVersion) => {
     if(review.index === latestVersion.index) {
-      return this.getVersionWithIcon(review, versions, latestVersion);
+      return getVersionWithIcon(review, versions, latestVersion);
     }
 
     return 'v' + review.index;
   }
 
-  getVersionWithIcon = (review, versions, latestVersion) => {
+  const getVersionWithIcon = (review, versions, latestVersion) => {
     if(versions === {})
```

Comment:
This comparison will always be false because {} creates a new object each time. Should be:

```suggestion
    if(Object.keys(versions).length === 0)
```